### PR TITLE
Doc page for Honeywell evohome CH/DHW controllers

### PR DESCRIPTION
**Pull request by zxdavb**
**Description:**
Basic documentation page for Honeywell evohome CH/DHW controller.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16427

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by DubhAd — APPROVED** [review]
👍

File: source/_components/climate.evohome.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: "Honeywell evohome CH/DHW Controller"
+description: "Instructions on how to integrate a Honeywell evohome controller with Home Assistant."
+date: 2018-09-25 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: honeywell.png
+ha_category: Climate
+ha_release: TBA (post 0.78)
```
Comment (MartinHjelmare) [inline source/_components/climate.evohome.markdown]: This isn't correct.

File: source/_components/evohome.markdown [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+description: "Instructions on how to integrate Honeywell evohome devices with Home Assistant."
+date: 2018-09-25 12:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: honeywell.png
-ha_category: Climate
+ha_category: Hub
 ha_release: TBA (post 0.78)
```
Comment (MartinHjelmare) [inline source/_components/evohome.markdown]: See above.

File: source/_components/evohome.markdown [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
 ha_iot_class: "Cloud Polling" 
 ---
 
-The `evohome` climate platform lets you control EU-based [Honeywell Connect Comfort](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW controllers from Home Assistant.  
+The `evohome` platform is the main component to set up and integrate all supported evohome devices.
```
Comment (MartinHjelmare) [inline source/_components/evohome.markdown:16]: This page is for the component, so it sounds weird to talk about platform.

**Pull request merged**